src/interaction/InteractionTrackingData.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * DisplayObjects with the {@link PIXI.interaction.interactiveTarget} mixin use this class to track interactions
+ *
+ * @class
+ * @private
+ * @memberof PIXI.interaction
+ */
+export default class InteractionTrackingData
+{
+    /**
+     * @param {number} pointerId - Unique pointer id of the event
+     */
+    constructor(pointerId)
+    {
+        this._pointerId = pointerId;
+        this._flags = this.constructor.FLAGS.NONE;
```

Comment:
I'd suggest using `InteractionTrackingData.FLAGS`, for one less variable depth, and in case someone does something weird like subclass InteractionTrackingData.